app/routes/tasks_routes.py

```diff
@@ -0,0 +1,134 @@
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, abort, make_response, request
+from datetime import *
+import requests
+import os
+
+
+tasks_bp = Blueprint("tasks_bp",__name__, url_prefix="/tasks")
+
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
```

Comment:
This is a powerhouse of a helper function! Since we use it just as frequently over in `goals_routes.py` and will probably use it in any additional routes files, I think it should live in its own helper file, like `routes_helpers.py`.